pkg/openstack/loadbalancer_test.go

```diff
@@ -985,12 +985,12 @@ func Test_buildPoolCreateOpt(t *testing.T) {
 					keepClientIP:         true,
 					tlsContainerRef:      "tls-container-ref",
 					proxyProtocolVersion: ptr.To(pools.ProtocolPROXY),
+					poolLbMethod:         "SOURCE_IP_PORT",
```

Comment:
can you leave these units tests as is, and just add new test cases, which use `poolLbMethod`?